### ISSUE TITLE
Swagger File invalid

### ISSUE THREAD

**Issue by jimmiebtlr** [akeneo/pim-api-docs#359]
Using this swagger tool it shows some invalid items (hidden in the bottom right of the page).  

https://validator.swagger.io/

The full message is 
```
{"messages":["attribute paths.'/api/rest/v1/asset-families/{asset_family_code}/attributes/{attribute_code}/options/{code}'(get).operationId is repeated","attribute paths.'/api/rest/v1/assets'(get).operationId is repeated","attribute paths.'/api/rest/v1/assets'(patch).operationId is repeated","attribute paths.'/api/rest/v1/assets/{code}'(get).operationId is repeated"]}
```

I'm seeing the same when I try to generate a go client using go-swagger, and having to go through and manually edit out the pieces causing havic.

The full error message from go-swagger seems clearer, and is 

```
The swagger spec at "/home/jimmiebtlr/ocm_microservices/akeno/akeno.swagger" is invalid against swagger specification 2.0. see errors :
- "patch_assets" is defined 2 times
- "get_assets" is defined 2 times
- "get_assets__code_" is defined 2 times
- "get_reference_entity_attributes__attribute_code__options__code_" is defined 2 times
```

**Comment by NolwennP** [comment]
Thank you for your feedback!
Here is the PR that will fix this: #362.

**Comment by jimmiebtlr** [comment]
That pr fixes it, thank you.
